ext/pdo_pgsql/pgsql_driver.stub.php

```diff
@@ -34,6 +34,7 @@ public function pgsqlGetNotify(int $fetchMode = PDO::FETCH_DEFAULT, int $timeout
     /** @tentative-return-type */
     public function pgsqlGetPid(): int {}
 
-    /** @tentative-return-type */
-    public function pgsqlSetNoticeCallback(?callable $callback): void {}
+    /* Do NOT add new methods here. See https://wiki.php.net/rfc/pdo_driver_specific_subclasses
```

Comment:
It's a nice way to solve the issue! :)

Reply:
Thank you! This reminder will help me remember how to avoid going through the whole process for the 173 next extensions I'm thinking about :-)